spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -231,6 +231,19 @@
         post :create, user: { email: user.email.upcase, password: user.password }
       end
 
+      it 'caches unverified PII pending confirmation' do
+        user = create(:user, :signed_up)
+        create(
+          :profile,
+          deactivation_reason: :verification_pending,
+          user: user, pii: { ssn: '1234' }
+        )
+
+        post :create, user: { email: user.email.upcase, password: user.password }
+
+        expect(controller.user_session[:decrypted_pii]).to match '1234'
```

Comment:
There's no metadata on the PII -- that's on the `Profile` record that the PII comes from. My expectation is that the `/profile` page would detect the `pending_profile` just like it does `active_profile` and route the user accordingly. That's for a future PR.